packages/component-library/src/CivicCard/CivicCardLayoutFull.js

```diff
@@ -1,13 +1,17 @@
-import React, { Fragment } from "react";
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import { Fragment } from "react";
```

Comment:
I wanted to use the `<> </>` Fragment syntax here, but in combination with the `/** @jsx jsx */` emotion syntax, I was having an issue.

If I recall if you `import React from "react"` it will work, but that line requires an `eslint ignore`

Reply:
Ah, I dug into it.

You actually have to import React from "react" for the `<> </>`  syntax to work, because [@babel/plugin-transform-react-jsx](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx) transforms it to React.fragment, so if you don't import React you will get an error.